src/layouts/FellowLayout.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Flex } from '@chakra-ui/react'
+
+import StarsImage from '@/public/images/stars.png'
+
+import ContentContainer from '@/components/ContentContainer'
+import { Image } from '@/components/Image'
+import FellowLayoutHero from '@/components/Heroes/FellowLayoutHero'
+import { MARKDOWN_CONTENT_MAX_WIDTH } from '@/utils/constants'
+import TableOfContents from '@/components/TableOfContents'
+import { H3 } from '@/components/Headings'
+
+export const FellowLayout = ({ children, frontmatter, tocItems }) => {
+  const { title, fellowName, country, tags, image } = frontmatter
+
+  return (
+    <>
+      <Box pos='absolute' zIndex={-1} top={-4} left={0} right={0}>
+        <Image src={StarsImage} alt="Stars" w='100%' />
```

Comment:
For this background images I'd recommend something like what [is recommended in the nextjs docs](https://nextjs.org/docs/app/building-your-application/optimizing/images#background-image):
```tsx
<Image
  src={StarsImage}
  alt="Stars"
  sizes="100vw"
  quality={100}
  style={{
    objectFit: "cover",
    width: "100%",
  }}
/>
```